pdb.mk

```diff
@@ -0,0 +1,54 @@
+PDB_HOME    := $(abspath $(BUILD_DIR)/xspdb)
+RTL_HOME    := $(abspath $(RTL_DIR))
+
+PICKER_INCLUDE := $(shell picker --show_xcom_lib_location_cpp | grep include | awk '{print $$2}')
+PYTHON_VERSION := $(shell python3 --version | awk '{print $$2}' | cut -d'.' -f1-2)
+SIM_LDFLAGS    := $(shell cat $(PDB_HOME)/python/sim_ld_flags.txt)
+
+PYTHON_DEPS = urwid capstone
+
+.PHONY: pdb swig-difftest xspython xspdb xspdb-compress pdb-run check-deps
+
+pdb: clean swig-difftest xspython xspdb
+
+swig-difftest: sim-verilog
+	$(MAKE) -C ./difftest difftest_python WITH_CONSTANTIN=0 WITH_CHISELDB=0
+
+xspython:
+	cp $(NOOP_HOME)/difftest/src/test/vsrc/common/* $(RTL_HOME)
+	cp $(PDB_HOME)/python/_difftest.so $(RTL_DIR)/libdifftest.so
+	find $(RTL_HOME) -maxdepth 1 \( -name "*.sv" -o -name "*.v" -o -name "*.cpp" -o -name "*.so" \) -not -name "SimTop.sv" -printf "%f\n" > $(RTL_HOME)/picker.f
+	time picker export $(RTL_HOME)/SimTop.sv \
+		--rw 1 \
+		-w $(PDB_HOME)/xs.fst \
+		--lang python \
+		--tdir $(PDB_HOME)/XSPython \
+		--fs $(RTL_HOME)/picker.f \
+		-V "--no-timing;--threads;8;+define+DIFFTEST;-I$(NOOP_HOME)/build/generated-src;" \
+		-C "-fPIC -lz -I$(PICKER_INCLUDE) -L$(RTL_HOME) -ldifftest -lpython$(PYTHON_VERSION) $(SIM_LDFLAGS)" \
+		--autobuild false 
+	cp $(RTL_HOME)/libdifftest.so $(PDB_HOME)/XSPython
+	$(MAKE) -C $(PDB_HOME)/XSPython
+
+xspdb:
+	cp $(PDB_HOME)/python/_difftest.so $(PDB_HOME)/XSPython
+	cp $(PDB_HOME)/python/difftest.py $(PDB_HOME)/XSPython
+	#ln -s $(PDB_HOME)/XSPython/_difftest.so $(PDB_HOME)/XSPython/libdifftest.so # it will use after picker update
+	cp -r $(NOOP_HOME)/scripts/xspdb $(PDB_HOME)/XSPdb
```

Comment:
try to use multiple PYTHONPATH in `pdb-run` instead of copy all things into single directory

Reply:
we should try to avoid repeatedly copying files here and there during the build process